### DISCUSSION TITLE
[Tooltip] aria-describedby not being read by voiceOver

### DISCUSSION THREAD

**Discussion by skarensmoll** [radix-ui/primitives#2037] [Help]
Hi there!

I'm using voiceover on my MacOS with the example provided on the documentation :

https://www.radix-ui.com/docs/primitives/components/tooltip

I was expecting the voice over would read `Add to library` text because the button is pointing to the Tooltips's content with the `aria-describedby` prop. However this was not the case, not sure why. 

I know this is intended to be used as additional content that's why I also included an aria-label to the button, and again I was expecting the voice over to read:  `this is an example, Add to library` but it didnt happen, I would appreciate some clarity on this.

```
<button  aria-label="this is an example" className="IconButton">
            <PlusIcon />
 </button>
```
 
 Thanks a lot in advance.

**Comment by benoitgrelard** [comment]
Hey @skarensmoll,

The way it is currently implemented, screen readers will announce what is in the content.
If you want to announce something different for screen readers to what is visually displayed in the tooltip content, you can pass `aria-label` on `Tooltip.Content`.

You can see an example of this here: https://601857eb614bae0021c9b9dd-zgkjtvnouj.chromatic.com/?path=/story/components-tooltip--aria-label
